internal/server/handlers.go

```diff
@@ -85,18 +85,43 @@ func (a *API) SignupRoute() route[api.SignupRequest, *api.SignupResponse] {
 	}
 }
 
+// Google returns the google social login provider
+func (a *API) Google(c *gin.Context) (*models.Provider, error) {
+	if a.server.options.GoogleClientID == "" {
+		return nil, nil // this is fine, the org just doesnt get google social login
+	}
+	google := &models.Provider{
+		Name:         "Google",
+		URL:          "accounts.google.com",
+		ClientID:     a.server.options.GoogleClientID,
+		ClientSecret: models.EncryptedAtRest(a.server.options.GoogleClientSecret),
+		CreatedBy:    models.CreatedBySystem,
+		Kind:         models.ProviderKindGoogle,
+		Managed:      true,
+	}
+	if err := a.setProviderInfoFromServer(c, google); err != nil {
+		return nil, err
+	}
```

Comment:
This is to set `AuthURL` and `Scopes`, is that right?

Does this always set the same value every time it's called, or would each org end up with different values for those fields?

Reply:
This moved to being statically set when we load the Google login from the server parameters